index.html

```diff
@@ -2,274 +2,295 @@
-                </p>
+      <div class="item">
+        <h3>Profesional Art Printing Data</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, molestiae quas vel sint commodi
+          repudiandae consequuntur voluptatum laborum numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga
+          praesentium optio, eaque rerum!</p>
 
-                <button type="submit" class="project-btn">Get in touch</button>
+        <ul class="apply-flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit" class="btn-recent-works 5">See Project</button>
+      </div>
+    </section> <!-- end work section -->
 
-            </form>
-        </section>
 
-        <section class="footer-section-end flex">
+    <!-- start about me section -->
+    <section class="about-section" id="about">
+      <div class="about-me-section">
+        <h2>About me</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
 
-            <ul class="flex">
-                <li><a href="">
-                        <a href="#"><img src="images/github.png" alt="git-icon" /></a>
-                    </a></li>
-                <li><a href="">
+        <div class="border-about-me-section"></div>
+      </div>
 
-                        <a href="#"><img src="images/linkedin.png" alt="linkden-icon" /></a>
-                    </a></li>
-                <li><a href="">
+      <div class="programming-languages apply-flex">
+        <div class="language">
+          <h4 class="languages">Language</h4>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+            <li></li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <h4 class="frames">Framworks</h4>
+          <ul>
+            <li>Bootstrap</li>
+            <li>RubyonRails</li>
+            <li>RSPec</li>
+            <li>CopyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <h4 class="my-skills">Skills</h4>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
 
-                        <a href="#"><img src="images/angellist.png" alt="angelist-icon" /></a>
-                    </a></li>
-                <li><a href="">
+    <section class="contact-section apply-flex" id="contact">
+      <h3>
+        I'm always interested in hearing about new projects, so if you'd like
```

Comment:
- [ ] kindly use the correct font color for desktop version of text... `#262626`